internal/kafka/internal/workers/kafka_mgrs/kafkas_routes_cname_mgr.go

```diff
@@ -18,12 +18,12 @@ type KafkaRoutesCNAMEManager struct {
 
 var _ workers.Worker = &KafkaRoutesCNAMEManager{}
 
-func NewKafkaCNAMEManager(kafkaService services.KafkaService, kafkfConfig *config.KafkaConfig, bus signalbus.SignalBus) *KafkaRoutesCNAMEManager {
+func NewKafkaCNAMEManager(kafkaService services.KafkaService, kafkfConfig *config.KafkaConfig, bus signalbus.SignalBus, reconciler workers.Reconciler) *KafkaRoutesCNAMEManager {
```

Comment:
```suggestion
func NewKafkaCNAMEManager(kafkaService services.KafkaService, kafkfConfig *config.KafkaConfig, reconciler workers.Reconciler) *KafkaRoutesCNAMEManager {
```

signal bus parameter can be removed throughout